examples/digital_fingerprinting/production/morpheus/dfp/modules/dfp_postprocessing.py

```diff
@@ -69,7 +69,7 @@ def on_data(message: MultiAEMessage):
 
         if logger.isEnabledFor(logging.DEBUG):
             logger.debug("Completed postprocessing for user %s in %s ms. Event count: %s. Start: %s, End: %s",
-                         message.meta.user_id,
+                         message.meta.df.username.iloc[0],
```

Comment:
Lets figure out why the `user_id` property is not set instead of using the dataframe directly